### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/IndexJoinOptimizer.java [hunk 1]
```diff
@@ -451,9 +452,10 @@ public void markSuccess()
             {
                 // Map from output Symbols to source Symbols
                 Map<Symbol, Symbol> directSymbolTranslationOutputMap = Maps.transformValues(Maps.filterValues(node.getAssignments(), SymbolReference.class::isInstance), Symbol::from);
-                Map<Symbol, Symbol> outputToSourceMap = FluentIterable.from(lookupSymbols)
-                        .filter(in(directSymbolTranslationOutputMap.keySet()))
-                        .toMap(Functions.forMap(directSymbolTranslationOutputMap));
+                Map<Symbol, Symbol> outputToSourceMap = lookupSymbols.stream()
+                        .filter(directSymbolTranslationOutputMap.keySet()::contains)
+                        .collect(toImmutableMap(Function.identity(), directSymbolTranslationOutputMap::get));
```
Comment (electrum) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/IndexJoinOptimizer.java]: Static import `identity`

File: presto-tests/src/main/java/com/facebook/presto/tests/tpch/TpchIndexedData.java [hunk 2]
```diff
@@ -67,9 +67,9 @@ public TpchIndexedData(String connectorId, TpchIndexSpec tpchIndexSpec)
             Map<String, ColumnHandle> columnHandles = new LinkedHashMap<>(tpchMetadata.getColumnHandles(null, tableHandle));
             for (Set<String> columnNames : tpchIndexSpec.getColumnIndexes(table)) {
                 List<String> keyColumnNames = ImmutableList.copyOf(columnNames); // Finalize the key order
-                Set<TpchScaledColumn> keyColumns = FluentIterable.from(keyColumnNames)
-                        .transform(TpchScaledColumn.columnFunction(table))
-                        .toSet();
+                Set<TpchScaledColumn> keyColumns = keyColumnNames.stream()
+                        .map(name -> new TpchScaledColumn(table, name))
+                        .collect(Collectors.toSet());
```
Comment (electrum) [inline presto-tests/src/main/java/com/facebook/presto/tests/tpch/TpchIndexedData.java]: Static import `toSet`. Why not use `toImmutableSet`?

Comment (electrum) [inline presto-tests/src/main/java/com/facebook/presto/tests/tpch/TpchIndexedData.java]: `FluentIterable.toSet()` returns `ImmutableSet`

**Comment by electrum** [comment]
Looks good

File: presto-tests/src/main/java/com/facebook/presto/tests/tpch/TpchIndexedData.java (hunk 2 above)
Comment (martint) [inline presto-tests/src/main/java/com/facebook/presto/tests/tpch/TpchIndexedData.java]: Ah, yes... I used Intellij's migrate intention and missed that one.

**Pull request closed without merging**